docs/guides/postinstall.md

```diff
@@ -113,3 +113,15 @@ If you still face an issue, mention it [here](https://github.com/t2linux/wiki/is
 The drivers for Wi-Fi and Bluetooth are included in a kernel with T2 support. But, we also need firmware to get them working from macOS.
 
 Instructions for the same are given in the [Wi-Fi and Bluetooth](https://wiki.t2linux.org/guides/wifi-bluetooth/) guide.
+
+# Network Manager recurrent notifications
+
+Some users has experience recurrent notifications due the internal usb ethernet interface connected to the T2 chip. To avoid those notifications we can blaklist `cdc_ncm` and `cdc_mbim` modules with the following command:-
+
+```sh
+sudo sh -c 'echo "# Disable for now T2 chip internal usb ethernet
+blacklist cdc_ncm
+blacklist cdc_mbim" >> /etc/modprobe.d/blacklist.conf
```

Comment:
I think the `'` quote never gets closed.

I think you need to either use add a `'` at the end or use `echo "foo" | sudo tee -a file` or `sudo sh -c "echo 'foo'